src/App.js

```diff
@@ -1,27 +1,7 @@
-import logo from './logo.svg';
-import './App.css';
+import Calculator from './components/Calculator';
 
 function App() {
```

Comment:
This component should be a class-based component. Consider updating it to the same.